.babelrc

```diff
@@ -2,9 +2,12 @@
   "plugins": [
     "transform-object-rest-spread",
     "transform-flow-comments",
+    "transform-runtime",
     "add-module-exports"
   ],
   "presets": [
-    "es2015"
+    "es2015",
```

Comment:
Whats the need for all of these transpilers?

I strongly prefer to use `babel-preset-env` with target set to whatever browser support.